models/user.go

```diff
@@ -1,25 +1,36 @@
 package models
 
+import "github.com/jinzhu/gorm"
+
 //UserFollowInfo holds the follow graph info, only relating userNames
 type UserFollowInfo struct {
-	UserName   string   `json:"user_name"`
-	RealName   string   `json:"real_name"`
-	AvatarURL  string   `json:"avatar_url"`
-	Bio        string   `json:"bio"`
-	Followers  []string `json:"followers"`
-	Followings []string `json:"followings"`
-	CrawlTs    int      `json:"crawl_ts"`
+	gorm.Model
+	UserName   string   `json:"user_name" gorm:"column:user_name"`
+	RealName   string   `json:"real_name" gorm:"column:real_name"`
+	AvatarURL  string   `json:"avatar_url" gorm:"column:avatar_url"`
+	Bio        string   `json:"bio" gorm:"column:bio"`
+	Followers  []string `json:"followers" gorm:"column:followers"`
+	Followings []string `json:"followings" gorm:"column:followings"`
+	CrawlTs    int      `json:"crawl_ts" gorm:"column:crawl_ts"`
 }
 
 // User is the struct containing all user fields, used for serializing users to dgraph
 type User struct {
+	gorm.Model
 	UID       string  `json:"uid,omitempty"`
-	Name      string  `json:"name,omitempty"`
-	RealName  string  `json:"real_name,omitempty"`
-	AvatarURL string  `json:"avatar_url,omitempty"`
-	Bio       string  `json:"bio,omitempty"`
-	Follows   []*User `json:"follows,omitempty"`
-	CrawledAt int     `json:"crawled_at,omitempty"`
+	Name      string  `json:"name,omitempty" gorm:"column:user_name"`
```

Comment:
I would suggest to keep it everywhere the same, so distinguish between UserName and RealName. In my opinion, just 'name' can be a bit confusing.